maven-plugin-api/pom.xml

```diff
@@ -92,6 +94,7 @@ under the License.
             </goals>
             <phase>pre-site</phase>
             <configuration>
+              <basedir>${basedir}</basedir>
```

Comment:
Should we also move the `plugin.mdo` along with others ?

Reply:
given it's not used with the Velocity-based generators, I think this would create even more confusion than the current situation where Velocity-based generation is used only on some .mdo but not all